Core/Config.cpp

```diff
@@ -443,9 +435,7 @@ static ConfigSetting graphicsSettings[] = {
 #endif
 
 	// TODO: Replace these settings with a list of options
-	ConfigSetting("PartialStretch", &g_Config.bPartialStretch, &DefaultPartialStretch, true, true),
```

Comment:
It might be nice to retain this default for Blackberry only; my understanding is that the aspect ratio of the display makes the zoom desirable as a default.  @xsacha?

-[Unknown]